includes/utils.php

```diff
@@ -1121,3 +1123,30 @@ function post_args_allow_list( $post_args ) {
 
 	return array_intersect_key( $post_args, array_flip( $allowed_post_keys ) );
 }
+
+/**
+ * Make a remote GET request.
+ *
+ * Wrapper function for wp_remote_get() and vip_safe_wp_remote_get(). The order
+ * of parameters differs from vip_safe_wp_remote_get() to promote the arguments array
+ * to the second parameter.
+ *
+ * See {@see http://developer.wordpress.org/reference/classes/WP_Http/request/ WP_Http::request} for $args defaults.
+ *
+ * @param  string $url       The URL to request.
+ * @param  array  $args      Optional. An array of arguments to pass to wp_remote_get()/vip_safe_wp_remote_get().
+ * @param  mixed  $fallback  Optional. Fallback value to return if the request fails. Default false. VIP only.
+ * @param  int    $threshold Optional. The number of fails required before subsequent requests automatically return the fallback value. Defaults to 3, with a maximum of 10. VIP only.
+ * @param  int    $timeout   Optional. The timeout for WP VIP requests. Default 3, 1-5 valid. VIP only, use $args for others.
```

Comment:
I believe only 1, 2 or 3 is valid here

Reply:
The time out maximum is complicated according to the code:
* 30 seconds for WP CLI requests
* 15 seconds for `POST` requests within the admin
* 5 seconds for front end and non POST request within the admin.

I'll expand the notes.
https://github.com/Automattic/vip-go-mu-plugins/blob/fdd99cf535e55eeda46d7a11a4cbb1603d8bc666/vip-helpers/vip-utils.php#L816-L831